roles/nova-common/defaults/main.yml

```diff
@@ -55,9 +55,9 @@ nova:
   vnc_keymap: en-us
   scheduler_default_filters: RetryFilter,AvailabilityZoneFilter,RamFilter,ComputeFilter,CoreFilter,ComputeCapabilitiesFilter,ImagePropertiesFilter,ServerGroupAntiAffinityFilter,ServerGroupAffinityFilter,AggregateInstanceExtraSpecsFilter,NUMATopologyFilter
   scheduler_host_subset_size: 1
-  libvirt_bin_version: 1.3.1-1ubuntu10.8~cloud0
+  libvirt_bin_version: 1.3.1-1ubuntu10.9~cloud0
   python_libvirt_version: 1.3.1-1ubuntu1~cloud0
-  qemu_kvm_version: 1:2.5+dfsg-5ubuntu10.9~cloud0
+  qemu_kvm_version: 2.0.0+dfsg-2ubuntu1.34
```

Comment:
update to this version(2.5+dfsg-5ubuntu10.13~cloud0)
https://apt-mirror.openstack.blueboxgrid.com/cloud_archive/ubuntu/pool/main/q/qemu/